x-pack/test/apm_api_integration/tests/alerts/rule_registry.ts

```diff
@@ -92,6 +96,13 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       .get(`/api/alerts/alert/${alert.id}`)
       .set('kbn-xsrf', 'foo');
 
+    const { body: targetIndices, status: targetIndicesStatus } = await getAlertsTargetIndices();
```

Comment:
what's the purpose of this exactly? This function only cares about whether the rule was executed. There might be a scenario where the alert index does not exist, but it's okay because no alerts were written.